src/OrleansGetEventStore/GetEventStoreProvider.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+using EventStore.ClientAPI;
```

Comment:
nvm, there is alpha release here https://www.nuget.org/packages/EventStore.ClientAPI.NetCore/